h2o-py/h2o/estimators/xgboost.py

```diff
@@ -742,6 +742,21 @@ def categorical_encoding(self, categorical_encoding):
         self._parms["categorical_encoding"] = categorical_encoding
 
 
+    @property
+    def te_model_id(self):
+        """
+        Key of TargetEncoderModel
+
+        Type: ``str``.
+        """
+        return self._parms.get("te_model_id")
+
+    @te_model_id.setter
+    def te_model_id(self, te_model_id):
+        assert_is_type(te_model_id, None, str, H2OEstimator)
```

Comment:
Need to fix xgboost manually as locally files are not being generated when xgboost is not available